drivers/i2s/i2s_sam_ssc.c

```diff
@@ -988,6 +987,7 @@ struct queue_item rx_0_ring_buf[CONFIG_I2S_SAM_SSC_RX_BLOCK_COUNT + 1];
 struct queue_item tx_0_ring_buf[CONFIG_I2S_SAM_SSC_TX_BLOCK_COUNT + 1];
 
 static struct i2s_sam_dev_data i2s0_sam_data = {
+	.dev_dma = DEVICE_DT_GET(DT_INST_DMAS_CTLR_BY_NAME(0, tx)),
```

Comment:
I'm not sure if it belongs in this PR but we could move this to `struct i2s_sam_dev_cfg`. I've made the change [i2s_sam_ssc.c](https://github.com/mnkp/zephyr/blob/i2s-sam-convert_dma/drivers/i2s/i2s_sam_ssc.c) and tested it on sam_e70_xplained board.

Reply:
I say do it as a follow up PR.